packages/flutter/test/material/progress_indicator_test.dart

```diff
@@ -251,6 +251,38 @@ void main() {
         ..rect(rect: const Rect.fromLTRB(0.0, 0.0, 200.0, 4.0))
         ..rect(rect: const Rect.fromLTRB(0.0, 0.0, 50.0, 4.0), color: primaryColor),
     );
+
+    // With ProgressIndicatorTheme colors
+    await tester.pumpWidget(
+      Theme(
+        data: ThemeData(
+          progressIndicatorTheme: const ProgressIndicatorThemeData(
+            color: Color(0xff0000ff),
```

Comment:
Could define this as a const color var `indicatorColor` and then test against `indicatorColor`.